spec/features/updating_event_spec.rb

```diff
@@ -124,12 +140,17 @@
     date = Date.tomorrow.in_time_zone.to_i * 1000
     page.execute_script("$('#event_start_date')\
                         .pickadate('picker').set('select', #{date})")
+    fill_in "event[start_time]", with: "09:00AM"
+
     page.execute_script("$('#event_end_date')\
                         .pickadate('picker').set('select', #{date})")
+    fill_in "event[end_time]", with: "02:00PM"
```

Comment:
Multiple instances of filling the same form. This can be extracted to it's own method which can then be easily called on a single command... something like `fill_in_form` or ....